### PR TITLE
[DE] Add `Beatmap_search`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Implements changes from #9828 

(Part 1 of 2)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap_search/de.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+| `title` | Songtitel |
+| `difficulty` | Name des Schwierigkeitsgrads der Beatmap |
+| `ar` | [Approach-Rate](/wiki/Beatmap/Approach_rate) |
+| `cs` | [Circle-Size](/wiki/Beatmap/Circle_size) |
+| `od` | [Allgemeine Schwierigkeit](/wiki/Beatmap/Overall_difficulty) |
+| `hp` | [HP Drain-Rate](/wiki/Beatmap/HP_drain_rate) |
+| `key`, `keys` | Anzahl der Keys (betrifft nur osu!mania und konvertierte Beatmaps) |
+| `star`, `stars` | [Sternebewertung](/wiki/Beatmap/Star_rating) |
+| `bpm` | [Songtempo](/wiki/Music_theory/Tempo) |
+| `length` | [Länge](/wiki/Beatmap/Play_time) in Sekunden |
```
Comment (The-Last-Cookie) [inline wiki/Beatmap_search/de.md:45]:
```suggestion
| `length` | [Dauer](/wiki/Beatmap/Play_time) in Sekunden |
```
Länge oder Dauer?

(Keep in mind this appears several times in this article)

File: wiki/Beatmap_search/de.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+stars>=7 stars<8 od<8
+```
+
+(Client) Zeige gerankte Beatmaps zum Thema Weihnachten an, die bisher nicht gespielt wurden:
+
+```
+unplayed= status=r christmas
+```
+
+(Client, Webseite) Prüfe, ob mindestens ein Level von ::{ flag=FR }:: [galvenize](https://osu.ppy.sh/users/381444) aus [DJ Fresh - Gold Dust](https://osu.ppy.sh/beatmapsets/28107) heruntergeladen wurde:
```
Comment (The-Last-Cookie) [inline wiki/Beatmap_search/de.md:103]: In a Discord server I'm in, many people expressed that the current word that is used for `difficulty` (Schwierigkeitsgrad) is never used by anyone and sounds too unwieldly. I was thinking that using `Level` more frequently might be a good alternative.

Before, I already have sometimes used this translation, though for transparency I'm noting it here.

File: wiki/Beatmap_search/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Beatmap_search/de.md:45]: I think Länge is fine

File: wiki/Beatmap_search/de.md (hunk 2 above)
Comment (respektive) [reply wiki/Beatmap_search/de.md:103]: I'm not really talking to a lot of people about osu! in German so I don't know what terms are actually more used than others, but level seems like a fine translation either way :+1:

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**